Items/04_01_Melee_&_Thrown_Weapons.txt

```diff
@@ -131,11 +129,10 @@ Range: Adjacent (1m)
 Accuracy Modifier: +1
 STR Requirement: 5
 
-	Considered concussive (explosive) damage. Hits armor points before physical 
-damage if the target has defense against explosive (i.e. concussive) damage. If 
-armor roll is less than 50%, target rolls a DC 50% fort save for a being stunned 
-for 1 turn. -30% stealth rolls for being quiet on stealth attacks. Reduces 
-door-breaking DC by 45% per hit. This weapon has a Guard modifier of 0.
+	If armor roll is less than 50, target must pass a DC 50 fort save 
+or be stunned for 1 turn. -30 stealth rolls for being quiet on stealth 

```

Comment:
Note to self: "-30 to Stealth checks"? Is Stealth a skill that I've specified? B/c if not I should do that. Also occurs above.

Reply:
https://github.com/bleehu/Compound_X/blob/dev/CharacterCreation/06_Skills.txt#L75

Reply:
Oh good! I... maybe need to use the sleeps.